packages/react-mutation-mapper/src/component/oncokb/OncoKbTreatmentTable.tsx

```diff
@@ -20,6 +20,7 @@ import SummaryWithRefs from './SummaryWithRefs';
 
 import mainStyles from './main.module.scss';
 import './oncoKbTreatmentTable.scss';
+import { EvidenceReferenceContent } from './oncokbCard/EvidenceReferenceContent';
```

Comment:
Very minor but can we move this above the style imports? And also while we are at it can we merge
```
import { LEVELS } from 'cbioportal-utils';
```
and
```
import { defaultArraySortMethod, defaultSortMethod } from 'cbioportal-utils';
```
into a single import statement?

Reply:
Weird they are not automatically updated. I will update.